docs/pages/reference/core/next-pages-router/index.mdx

```diff
@@ -0,0 +1,62 @@
+# Next.js Pages Router integration
+
+Frames.js can be easily integrated with [Next.js](https://nextjs.org) application that uses [Pages Router](https://nextjs.org/docs/pages).
+
+## Create Frames handler
+
+Frames handler is responsible for rendering your Frames and also reacts to user interactions with buttons.
+
+```tsx [./pages/api/frames/index.tsx]
+/* eslint-disable react/jsx-key */
+import { createFrames, Button } from "frames.js/next/pages-router";
+
+const frames = createFrames({
+  basePath: "/api/frames",
+});
```

Comment:
i think this may cause a compilation error if it's not in a separate file

Reply:
I don't see a reason why it should.

Reply:
me neither, but was not that the issue in the framesjs-starter example?

Reply:
The issue was when you exported that `export const frames = createFrames()` and the issue was only in app router if I remember correctly.

I tried to build it locally and it builds successfully.